packages/extension-store-locator/jest.config.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const base = require('@salesforce/pwa-kit-dev/configs/jest/jest.config.js')
+
+module.exports = {
+    ...base,
+    moduleNameMapper: {
```

Comment:
What is this?

Reply:
jest don't know where to look at file import path that contains the wild card, like

```
import { withStoreLocator } from "*/components/store-locator/with-store-locator";
```

This is to tell Jest where the actual file is.